src/root/views/Country/Preparedness/EapApplication/EapOverview/styles.module.scss

```diff
@@ -0,0 +1,46 @@
+@import '~styles/settings/_variables.scss';
+
+.eap-overview {
+  :global {
+    .go-input-section {
+      width: 100%;
+    }
+  }
+
+  .eap-overview-container {
+    background-color: $white;
+  }
+
+  .eap-country {
+    display: flex;
+  }
+
+  .eap-disaster {
+    width: 50%;
+  }
+}
+
+.urban-to-rural {
+  .inputs {
+    display: flex;
+  }
+}
+
+.key-partners {
+  display: flex;
+}
+
+.actions {
+  flex-basis: unset;
+  flex-grow: unset;
+
+  .add-key-button {
+    display: flex;
+    justify-content: flex-end;
+    margin-top: $spacing;
```

Comment:
use symmetric design if possible. Consult with @frozenhelium once if this can be avoided,